craft_providers/lxd/launcher.py

```diff
@@ -755,6 +755,13 @@ def launch(
         )
         return instance
 
+    # check if the base instance is still being creating
```

Comment:
```suggestion
    # check if the base instance is still being created
```

Sorry for suggesting a typo!  We can just leave this as-is so we don't have to wait on CI